{"name": "test_vwo_login_negative_tc3", "status": "broken", "statusDetails": {"message": "_pytest.fixtures.FixtureLookupError: ('setup', <FixtureRequest for <Function test_vwo_login_negative_tc3>>)", "trace": "file C:\\Python-Selenium\\Framework\\Automation_Framework2\\tests\\test\\vwoLoginTests\\prod\\test_vwo_login_pf.py, line 51\n      @pytest.mark.usefixtures(\"setup\")\n      @pytest.mark.qa\n      def test_vwo_login_negative_tc3(self, setup):\nE       fixture 'setup' not found\n>       available fixtures: _session_faker, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, doctest_namespace, extra, extras, faker, include_metadata_in_junit_xml, metadata, monkeypatch, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, testrun_uid, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, worker_id\n>       use 'pytest --fixtures [testpath]' for help on them.\n\nC:\\Python-Selenium\\Framework\\Automation_Framework2\\tests\\test\\vwoLoginTests\\prod\\test_vwo_login_pf.py:51"}, "start": 1741009204896, "stop": 1741009204896, "uuid": "eddbccf3-d622-4430-80a5-03e4522e7f6a", "historyId": "a49ac24ea773ebb9f62986076f9f6e91", "testCaseId": "a49ac24ea773ebb9f62986076f9f6e91", "fullName": "tests.test.vwoLoginTests.prod.test_vwo_login_pf.TestVWOLogin#test_vwo_login_negative_tc3", "labels": [{"name": "epic", "value": "VWO App"}, {"name": "feature", "value": "Login Test"}, {"name": "tag", "value": "qa"}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup')"}, {"name": "parentSuite", "value": "tests.test.vwoLoginTests.prod"}, {"name": "suite", "value": "test_vwo_login_pf"}, {"name": "subSuite", "value": "TestVWOLogin"}, {"name": "host", "value": "DESKTOP-SJ9H8P2"}, {"name": "thread", "value": "6812-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test.vwoLoginTests.prod.test_vwo_login_pf"}]}